mmp_to_musicxml/converter.py

```diff
@@ -203,16 +203,21 @@ class MMP_MusicXML_Converter:
 	
 	SPECIFIED_KEY_SIGNATURE = None
 	
-	def __init__(self, check_notes=False, key_signature=None):
+	def __init__ ( self, key_signature = None, params = None ):
```

Comment:
can we remove the spaces after and before the parentheses + remove the spaces around the equals signs?